azext_edge/edge/_help.py

```diff
@@ -115,7 +115,7 @@ def load_iotops_help():
             - {COMPAT_OPCUA_APIS.as_str()}
             - {COMPAT_DATAFLOW_APIS.as_str()}
 
-            For more information on cluster requirements, please check https://aka.ms/iot-ops-cluster-requirements
```

Comment:
Please leave in the `https://` protocol, otherwise these may not be interpreted by some terminals as links

Reply:
kk FYI @vilit1

Reply:
oh uhhhh *looks at the other aka.ms links

tbh it is all just letters in the end

Reply:
yeah looks like it's inconsistent across this file, I see `aka.ms/lakehouse-delta-sample` `aka.ms/aziotops-arcconnect`also without https, but the ones I removed do have https, can we decide on wether we should have or not?

Reply:
later pr then